pkgs/development/python-modules/autoflake/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi, pyflakes }:
+
+buildPythonPackage rec {
```

Comment:
This does not look a library to me. If so, can you use `buildPythonApplication` and move it to `all-packages.nix`?